python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -0,0 +1,637 @@
+        self.psf_schema = afwTable.SourceTable.makeMinimalSchema()
+        self.makeSubtask("psf_detection", schema=self.psf_schema)
+        self.makeSubtask("psf_measurement", schema=self.psf_schema)
+        self.makeSubtask("psf_measure_psf", schema=self.psf_schema)
+
+        self.makeSubtask("measure_aperture_correction", schema=self.psf_schema)
+
+        # star measurement subtasks
+        if output_schema is None:
+            output_schema = afwTable.SourceTable.makeMinimalSchema()
+        self.makeSubtask("star_detection", schema=output_schema)
+        self.makeSubtask("star_deblend", schema=output_schema)
+        self.makeSubtask("star_measurement", schema=output_schema)
+        self.makeSubtask("star_apply_aperture_correction", schema=output_schema)
+        self.makeSubtask("star_catalog_calculation", schema=output_schema)
+        self.makeSubtask("star_set_primary_flags", schema=output_schema, isSingleFrame=True)
+        self.makeSubtask("star_selector")
+
+        self.makeSubtask("astrometry", schema=output_schema)
+        self.makeSubtask("photometry", schema=output_schema)
+
+        self.makeSubtask("compute_summary_stats")
+
+        # For the butler to persist it.
+        self.output_schema = afwTable.SourceCatalog(output_schema)
+
+    def runQuantum(self, butlerQC, inputRefs, outputRefs):
+        inputs = butlerQC.get(inputRefs)
+
+        astrometry_loader = lsst.meas.algorithms.ReferenceObjectLoader(
+            dataIds=[ref.datasetRef.dataId for ref in inputRefs.astrometry_ref_cat],
+            refCats=inputs.pop("astrometry_ref_cat"),
+            name=self.config.connections.astrometry_ref_cat,
+            config=self.config.astrometry_ref_loader, log=self.log)
+        self.astrometry.setRefObjLoader(astrometry_loader)
+
+        photometry_loader = lsst.meas.algorithms.ReferenceObjectLoader(
+            dataIds=[ref.datasetRef.dataId for ref in inputRefs.photometry_ref_cat],
+            refCats=inputs.pop("photometry_ref_cat"),
+            name=self.config.connections.photometry_ref_cat,
+            config=self.config.photometry_ref_loader, log=self.log)
+        self.photometry.match.setRefObjLoader(photometry_loader)
+
+        outputs = self.run(**inputs)
+
+        butlerQC.put(outputs, outputRefs)
+
+    def run(self, *, exposure):
+        """Find stars and perform psf measurement, then do a deeper detection
+        and measurement and calibrate astrometry and photometry from that.
+
+        Parameters
+        ----------
+        exposure : `lsst.afw.image.Exposure`
+            Post-ISR exposure, with an initial WCS, VisitInfo, and Filter.
+            Modified in-place during processing.
+
+        Returns
+        -------
+        result : `lsst.pipe.base.Struct`
+            Results as a struct with attributes:
+
+            ``output_exposure``
+                Calibrated exposure, with pixels in nJy units.
+                (`lsst.afw.image.Exposure`)
```

Comment:
No, it does, just one that is identically 1. That way people can use it as they normally would, it just does nothing. I think that's better than not including a photoCalib, because that's not the same thing as saying the image is calibrated (e.g., a `raw`).